dataquality/loggers/logger_config/seq2seq/encoder_decoder.py

```diff
@@ -0,0 +1,10 @@
+from dataquality.loggers.logger_config.seq2seq.seq2seq_base import Seq2SeqLoggerConfig
+
+
+class EncoderDecoderLoggerConfig(Seq2SeqLoggerConfig):
+    # TODO Add comment
+    # This currently is purely a wrapper!
+    pass
```

Comment:
```suggestion
    """Logger config for Encoder Decoder
    
    This logger currently has same fields as the base class
    """
```

something like this is fine ^ also you don't need the "pass"